client/.env.template

```diff
@@ -0,0 +1,3 @@
+REACT_APP_INFURA_URI=
+REACT_APP_CHAIN_ID=
+REACT_APP_BANK_CONTRACT_ADDRESS=
```

Comment:
This doesn't get used

Reply:
a .env is used anywhere you see process.env.

Will include instructions in the readme when this is out of WIP phase.